line_obfuscator.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"fmt"
+	"go/ast"
+	mathrand "math/rand"
+	"strings"
+
+	"golang.org/x/tools/go/ast/astutil"
+)
+
+const (
+	// PosMax is the largest line or column value that can be represented without loss.
+	// Source: https://golang.org/src/cmd/compile/internal/syntax/pos.go
```

Comment:
nit: make both source links consistent. I would personally go for https://go.googlesource.com/go/+/refs/heads/master/src/, since you can link to files at specific commits (like github), but you don't have to rely on the github mirror.